lib/internal/streams/end-of-stream.js

```diff
@@ -8,7 +8,8 @@ const {
   codes,
 } = require('internal/errors');
 const {
-  ERR_STREAM_PREMATURE_CLOSE
+  ERR_STREAM_PREMATURE_CLOSE,
+  ERR_INVALID_ARG_TYPE
```

Comment:
nit: prefer alphabetical order here please.
```suggestion
  ERR_INVALID_ARG_TYPE,
  ERR_STREAM_PREMATURE_CLOSE,
```

Reply:
Perhaps we can enforce this with the linter - https://eslint.org/docs/rules/sort-keys, wdyt?

Reply:
Great suggesstion.

Reply:
https://github.com/nodejs/node/issues/41767